lib/datasource/rubygems/get.ts

```diff
@@ -63,13 +88,15 @@ export async function getDependency(
     }
   }
 
+  // FIXME: invalid properties for `Release`
```

Comment:
Let's create an issue for this and with more of a description. I'd prefer it to be a TODO too, assuming it's not an urgent fix

Reply:
Created https://github.com/renovatebot/renovate/issues/11312